qemu/tests/cpu_hv_flags_smoke.py

```diff
@@ -0,0 +1,34 @@
+from virttest import error_context
+from provider import qemu_img_utils
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    Add 'hv-enforce-cpuid' option to see whether guest use features we didn't advertize to it.
+    1. Boot the guest with cpu_model_flags.
+    2. login
+    param test: the test object
+    param params: the test params
+    param env: the test env object
+    """
+
+    res = []
+    cpu_model_flags_list = params.objects("cpu_model_flags_list")
+    for cpu_model_flags in cpu_model_flags_list:
+        try:
+            params["cpu_model_flags"] = cpu_model_flags
+            vm = qemu_img_utils.boot_vm_with_images(test, params, env)
+            vm.wait_for_login(timeout=360)
+        except Exception:
+            res.append("The cpu hyperV flags case in failure: %s. " % cpu_model_flags)
+            pass
+        finally:
+            if vm:
+                vm.graceful_shutdown(timeout=360)
+                vm.destroy()
+
+    if len(res) > 0:
+        for case in res:
+            test.log.debug(case)
+        test.fail("The number of failed cases is: %s. " % len(res))
```

Comment:
could we also print the value of res ? This will make the result more clear to see which case failed.

Reply:
could we also print the value of res ? This will make the result more clear to see which case failed. ---> the res has been print in "for" loop.  
` 
for case in res:    
  test.log.debug(case)  
`